ipatests/test_integration/test_sso.py

```diff
@@ -53,6 +54,40 @@ def keycloak_login(host, username, password, username_fl=None):
         host.run_command(["rm", "-f", "/tmp/keycloak_login.py"])
 
 
+def keycloak_add_user(host, kcadm_pass, username, password=None,
+                      username_fl=None):
+    if username_fl is None:
+        username_fl = username
```

Comment:
What is the _fl option for, and is what does the "fl" stand for? It appears to be unused.

Reply:
fl stands for "first last".  The intention was to include that info when creating the user in keycloak.  However, that didn't turn out to be necessary yet.  I'm going to run a quick test to see if I can easily add that code so we have it for later if needed.  If I can't, I'll remove this.

Reply:
I removed this.  For this test it's not needed and it was going to be more complicated than previously thought.